config.js

```diff
@@ -1,4 +1,10 @@
 'use strict';
+try {
+  // eslint-disable-next-line implicit-dependencies/no-implicit
```

Comment:
The implicit dependencies rule isn't used in this repo so there's no need for the ignore comment
```suggestion
```